### PR TITLE
Add setup instructions to readme

### PR DESCRIPTION
**Pull request by caward12**
Added ruby and rails versions 
Added setup instructions for those who would like to contribute to the project.

### CODE REVIEW

**Comment by caward12** [comment]
Hello - I found this project via Ruby For Good's website and saw the slim documentation for those interested in contributing so I added some simple set up instructions to the readme. Please let me know if I should add anything else!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+`
+cd scan-donation
+`
+
+`
+bundle
+`
+
+`
+rake db:create
```
Comment (kwokster10) [inline README.md]: Would you inline the rake commands `rake db:create db:migrate`?

**Comment by kwokster10** [comment]
@caward12 Thanks for adding the setup instructions to the README. I added a couple of comments.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
 # scan-donation
 Donation app for Stop Child Abuse Now of Northern VA
+
+
+## Setup
+This project uses
+
+ruby `2.4.1`
```
Comment (kwokster10) [inline README.md]: Can you capitalize `ruby` and `rails` on L10? Thanks.

**Comment by caward12** [comment]
@kwokster10 I made the updates you suggested. Let me know if there is anything else that needs updating!

**Pull request merged**